src/NuGet.Services.Validation/Entities/CvsScanOperationState.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
```

Comment:
Please make sure to add our file header with the license information. For example:

https://github.com/NuGet/NuGet.Jobs/blob/a06b44a7e1d26b194d36f8dc15e381bba92da90b/src/Ng/Program.cs#L1-L3

```suggestion
// Copyright (c) .NET Foundation. All rights reserved.
// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.

﻿using System;
```

Reply:
Updated.